src/pages/generate/constants/curationDetectionMode.ts

```
@@ -0,0 +1,21 @@
+export type CurationDetectionMode = 'server' | 'client';
+
+export const CURATION_DETECTION_MODE =
```

ESLint

[error] Insert `·(`

(prettier/prettier)